pkg/tsdbctl/check.go

```diff
@@ -230,16 +230,30 @@ func (cc *checkCommandeer) printResponse(resp *v3io.Response) error {
 
 			if values != nil {
 				bytes := values.([]byte)
-				err := cc.printValues(bytes)
-				if err != nil {
-					return err
+				if strings.HasPrefix(attr, "_v_") {
+					cc.printArrays(bytes)
+				} else {
+					err := cc.printValues(bytes)
+					if err != nil {
+						return err
+					}
 				}
 			}
 		}
 	}
 	return nil
 }
 
+func (cc *checkCommandeer) printArrays(bytes []byte) {
+	fmt.Print("[")
```

Comment:
How about building one string and print it once instead of multiple `fmt.Print` calls?

Reply:
+1